dc-model-feign/src/main/java/de/digitalcollections/model/feign/encoder/paging/PageRequestToQueryMapEncoder.java

```diff
@@ -0,0 +1,34 @@
+package de.digitalcollections.model.feign.encoder.paging;
+
+import de.digitalcollections.model.api.paging.PageRequest;
+import de.digitalcollections.model.feign.encoder.filter.FilteringToQueryMapEncoder;
+import feign.QueryMapEncoder;
+import java.util.HashMap;
+import java.util.Map;
+
+public class PageRequestToQueryMapEncoder implements QueryMapEncoder {
```

Comment:
Docstring wäre gut, für was ist der gut, wie wird er verwendet? Finde hier im Code oder im PR allgemein leider nichts dazu.

Reply:
War im PR der Satz: "Add feign module containing helper-classes for serializing PageRequest to rest-urls (analog to jackson and xml module)"
ich ergänz noch javadoc

Reply:
done. auch README.md hinzugefügt